pkg/sql/parser/datum.go

```diff
@@ -38,10 +38,15 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror"
 	"github.com/cockroachdb/cockroach/pkg/util/duration"
 	"github.com/cockroachdb/cockroach/pkg/util/ipaddr"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/cockroach/pkg/util/uint128"
 	"github.com/cockroachdb/cockroach/pkg/util/uuid"
 )
 
+func init() {
+	time.Local = time.UTC
```

Comment:
um

Reply:
heh, oh, yeah, that was what this commit was supposed to avoid.